packages/communication-react/package.json

```diff
@@ -62,6 +62,8 @@
     "test": "",
     "test:coverage": "",
     "check-deps": "node scripts/verifyDependencies",
+    "check-types-field": "node scripts/verifyTypeTrimmedDeclaration verify",
```

Comment:
nitty nit: A more contextual name here will help (`build` builds the package, `test` tests the package, `check-types-field` type checks the package (?!))

How about `validate-package-json` and `update-package-json` (it can handle the `types` field, or other automation we might need in the future (e.g., tracking different versions of the headless SDKs))

Reply:
Leaving a bit specific for now (as `validate-package-json-types` and `update-package-json-types`) and will let you alter it slightly in the future. Reason being is we have a `check-deps` that also is a check on the package.json, but this runs at build time instead of just in a branch.

Reply:
Actually - perhaps ours should also run at build time, and should use the package version to dictate what to check for. i.e. if its 1.0.0-beta.7 it should validate the types field is a beta reference.